migrations/2023-06-15-add-tile.js

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env node
+/* eslint no-await-in-loop: off */
+require('dotenv').config()
+const mongo = require('../lib/util/mongo')
+const {calcMetaTilesNumero, calcMetaTilesVoie} = require('../lib/util/tiles')
+
+async function addTilesToNumeros() {
+
+  const numeros = await mongo.db.collection('numeros')
+    .find({_deleted: null})
+    .project({_id: 1, positions: 1})
+    .toArray()
+
+  return Promise.all(
+    numeros
+      .filter(n => n.positions)
+      .map(n => {
+        n = calcMetaTilesNumero(n)
+        return mongo.db.collection('numeros').updateOne({_id : n._id}, {$set: n})
+      })
+  )
+}
+
+async function addTilesToVoies() {
+  const voies = await mongo.db.collection('voies')
+    .find({_deleted: null})
+    .project({_id: 1, typeNumerotation: 1, trace: 1})
+    .toArray()
```

Comment:
Pour éviter de charger toutes les voies en mémoire on pourrait pas utiliser un curseur pour l'itération?